src/psyclone/psyir/nodes/omp_directives.py

```diff
@@ -138,28 +139,30 @@ def node_str(self, colour=True):
         '''
         Returns the name of this node with (optional) control codes
         to generate coloured output in a terminal that supports it.
-        :param bool colour: whether or not to include colour control codes.
+
+        :param bool colour: Whether or not to include colour control codes.
```

Comment:
(picky) but lowercase to start the parameter description please and this line is also missing a line-continuation symbol (`\`)

Reply:
Done